homeassistant/components/omnilogic/switch.py

```diff
@@ -121,7 +121,7 @@ async def async_turn_off(self, **kwargs):
         """Turn off the relay."""
         self._state = False
         self._last_action = time.time()
-        self.async_schedule_update_ha_state()
+        self.async_write_ha_state()
```

Comment:
Please replace all occurances of `async_schedule_update_ha_state`.

Reply:
Addressed - sorry, missed the multiple occurrences!